scripts/aem.js

```diff
@@ -274,13 +274,15 @@ function getMetadata(name, doc = document) {
  * @param {string} [alt] The image alternative text
  * @param {boolean} [eager] Set loading attribute to eager
  * @param {Array} [breakpoints] Breakpoints and corresponding params (eg. width)
+ * @param {string} height Default height of the image
```

Comment:
Shouldn't need to change anything in `aem.js`

Reply:
I removed the changes, thanks for pointing that